scripts/size_report/size_report_helper.ts

```diff
@@ -15,61 +15,20 @@
  * limitations under the License.
  */
 
-import {
-  generateReportForModules,
-  Report
-} from '../repo-scripts/size-analysis/analysis-helper';
-
 import { execSync } from 'child_process';
 import * as https from 'https';
-import { mapWorkspaceToPackages } from './release/utils/workspace';
-import { projectRoot } from './utils';
 
-interface RequestBody {
+export interface RequestBody {
   log: string;
-  modules: Report[];
 }
-const runId: string = process.env.GITHUB_RUN_ID || 'local-run-id';
-
-const METRICS_SERVICE_URL: string = process.env.METRICS_SERVICE_URL!;
+export const runId = process.env.GITHUB_RUN_ID || 'local-run-id';
 
-async function generateReport(): Promise<RequestBody> {
-  const requestBody: RequestBody = {
-    log: `https://github.com/${process.env.GITHUB_REPOSITORY}/actions/runs/${runId}`,
-    modules: []
-  };
-  try {
-    let allModulesLocation: string[] = await mapWorkspaceToPackages([
-      `${projectRoot}/packages-exp/*`
-    ]);
+export const METRICS_SERVICE_URL = process.env.METRICS_SERVICE_URL!;
```

Comment:
Do we need to export `METRICS_SERVICE_URL`? Or `constructRequestOptions`, `constructRequestPath`?